src/pages/markets/components/markets/_synthetic-indices.tsx

```diff
@@ -59,6 +62,29 @@ const StockIndices = ({ simple_step_content }: StockIndicesProps) => {
                 }
                 display_title={<Localize translate_text="Synthetics trades available on Deriv" />}
             />
+            <Flex.Box
+                direction="col"
+                container="fluid"
+                justify="center"
+                align="center"
+                pb="10x"
+                md={{ pb: '40x', mb: '20x' }}
+            >
+                <Typography.Paragraph size="medium" mb="10x" textcolor="black" align="center">
+                    <Localize translate_text="_t_Want to know more about CFD trading conditions for the instruments we offer?_t_" />
+                </Typography.Paragraph>
+                <LinkButton.Primary
+                    font_family="UBUNTU"
+                    size="medium"
+                    aria-label="check trading specs"
+                    url={{
+                        type: 'internal',
+                        to: '/trading-specification',
+                    }}
+                >
+                    <Localize translate_text="_t_Check trading specs_t_" />
+                </LinkButton.Primary>
```

Comment:
@konstantinosG-deriv just asking if its same block of code so can we make it a component. Or maybe we can pick this later for improvement.

Reply:
It's the same block so I guess it would be a good idea in the future to make a component out of it 👍